udata/core/site/metrics.py

```diff
@@ -114,7 +114,7 @@ class MaxDatasetFollowersMetric(SiteMetric):
     def get_value(self):
         dataset = (Dataset.objects(metrics__followers__gt=0).visible()
                           .order_by('-metrics.followers').first())
-        return dataset.metrics.get('followers', 0)
+        return dataset.metrics.get('followers', 0) if dataset else 0
```

Comment:
Your fix is right but I believe that `metrics_followers__gt=0` ensure there is always a `followers` metric if there is a matching dataset so you can write:
```python
return dataset.metrics['followers'] if dataset else 0
```